src/Core/src/Handlers/Entry/EntryHandler.Android.cs

```diff
@@ -68,8 +64,10 @@ protected override void DisconnectHandler(AppCompatEditText platformView)
 			_set = false;
 		}
 
-		public static void MapBackground(IEntryHandler handler, IEntry entry) =>
+		public static void MapBackground(IEntryHandler handler, IEntry entry)
```

Comment:
Because this file doesn't have any functional changes can we leave this file unchanged?